examples/2d/contributors.rs

```diff
@@ -52,19 +61,22 @@ const ALPHA: f32 = 0.92;
 
 const SHOWCASE_TIMER_SECS: f32 = 3.0;
 
-fn setup(
+const CONTRIBUTORS_LIST: &[&str] = &["Carter Anderson", "And Many More"];
+
+fn setup_contributor_selection(
     mut commands: Commands,
     asset_server: Res<AssetServer>,
     mut materials: ResMut<Assets<ColorMaterial>>,
 ) {
-    let contribs = contributors();
+    // Load contributors from the git history log or use default values from
+    // the constant array. Contributors must be unique, so they are stored in a BTreeSet
```

Comment:
If the primary/only reason for the chosen Data structure is forcing uniqueness, wouldn't `HashSet` be a better choice here for beginners?

Reply:
@alice-i-cecile What do you think about the order of contributors' names? We are shuffling them later.
`contributor_selection.order.shuffle(&mut rnd);`

Reply:
I don't have strong feelings on the ordering. I agree with the comment about `HashSet`, but don't particularly mind. If you do use `HashSet`, grab the (faster, insecure) version from `bevy_utils`.

Reply:
Replaced `BTreeSet` with a `HashSet`. Also updated the docs and used `.collect()` instead of `from_iter()`